app/services/candidate_interface/continuous_applications/submit_application_choice.rb

```diff
@@ -15,7 +15,7 @@ def call
 
         ActiveRecord::Base.transaction do
           application_choice.assign_attributes(personal_statement: application_form.becoming_a_teacher)
-          application_form.update!(submitted_at:)
+          application_form.update!(submitted_at:) if application_form.application_choices.one?
```

Comment:
If the user create 4 drafts and submit one the submitted_at won't be updated. 🤔 🤔 🤔  Maybe we need to check the submissions are zero? `application_form.submitted_choices.zero?`. Or just check the submitted at is nil? `application_form. submitted_at.blank?`

Reply:
I think we need both, just one submitted choice. `submitted_at.blank?` will be true for a draft as well.

Reply:
I deleted my previous comment. Yeah, you need to check that there are no submitted choices. `submitted_at.blank?` on its own won't work for the first time you want to update.